hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/contracts/ParsingConstants.java

```diff
@@ -41,35 +41,16 @@ private ParsingConstants() {
 
     // struct types
     public static final String EXPIRY = "(uint32,address,uint32)";
```

Comment:
It's probably my confusion but why are the non versioned constants (e.g. EXPIRY) still here?  I see the *_struct_V1 versions in SystemContractsAbis.  Should not all references be move to there?

Reply:
["It's a process, it's a process..."](https://www.youtube.com/watch?v=5AGY0Y1OiF4&t=65s)

I started with the _versioned_ methods first, because that was the chance for the greatest inconsistencies between code here and solidity truth.  But the rest are coming.  `ParsingConstants` is for the chopping block.